hack/shell-bashrc

```diff
@@ -1,4 +1,7 @@
 # Only load `~/.bashrc` in interactive shells
+if [[ ! -z "$TARGET_DIR" ]]; then
```

Comment:
@NicolasT: I did this at first try to solve the CI failing. I didn't remove this because I found convenient to not change ANSIBLE_INVENTORY (without, we need to patch the ANSIBLE_INVENTORY which might be a relative path evaluated from pwd (that change running under a make shell for an extracted archive)).

This requires to patch the 1.0.0 version (shutil.copy in create_version_archive), which is a bit ugly...
What do you think about that

Reply:
First of all, needs to be moved down (the comment above applies to the section below).
I don't really like this to be part of a shell rcfile... Must be a better way to achieve this... Not sure how though right now.